docs/language/expressions.md

```diff
@@ -548,7 +546,7 @@ yield under(this)
 Union values are powerful because they provide a mechanism to precisely
 describe the type of any nested, semi-structured value composed of elements
 of different types.  For example, the type of the value `[1,"foo"]` in JavaScript
-is simply a generic JavaScript "object".  But in Zed, the type of this
+is simply a generic JavaScript "object".  But in SuperSQL, the type of this
```

Comment:
I paused for a bit here unsure if I should say Super (the format, since the value `[1,"foo"]` we just showed can be interpreted as a value in the Super format) or SuperSQL (the language, since `[1,"foo"]` can also be interpreted as an array literal in the language). Since this doc is technically part of the Language docs I went with the latter, though I'd be open to mentioning both if that's what folks prefer.